MakingMusicTool/MakingMusicTool.py

```diff
@@ -93,65 +90,48 @@ def analyse_data(self, fileName:str)->dict:
         dict
             sound pattern of the csv file
         """
-        # make absolute path
-        path = os.path.dirname(os.path.abspath(__file__))
-        abs_path = os.path.join(path, fileName)
+        absolute_path = get_absolute_path(fileName)
         sound_pattern = dict()
-        label = "sound"
         #open csv file
-        with open(abs_path, newline='') as file:
+        with open(absolute_path, newline='') as file:
             reader = csv.reader(file)
             # iterate through file input
             for i,row in enumerate(reader):
                 # delete all empty strings
                 row = [x for x in row if x]
-                # check if notes are correct defined
-                if self.__check_notes(row):
+                if self.__are_notes_valid(row[2:]):
                     # add to dict
-                    sound_pattern[label+str(i)]=row
+                    sound_pattern["sound"+str(i)] = row
                 else:
                     #TODO do something if notes aren't valid
                     print(f"Check the pattern of row {i}")
         return sound_pattern
     
     
-    def makeMusic(self):
+    def make_music(self):
         """
         Use the imported music information to create a soundtrack.
         """
-        # silent audio segment with defined length for one 4/4 time
-        music = AudioSegment.silent(duration=self.__44time_duration)
+        music = AudioSegment.silent(duration=self.__time_signature_duration)
         # iterating though the importet patterns
         for pattern in self.sound_pattern.values():
-            # get instrument sound
-            sound = self.__choose_instrument(pattern[0])
-            # set instrument sound to defined volume
-            sound = self.__sync_target_amplitute(sound, self.__ref_dBFS)
-            # get the choosen volume
-            volume = self.__choose_volume(pattern[1])
-            # increase the volume of the audio segment
+            sound = self.__get_instrument_sound(pattern[0])
+            sound = self.__set_volume(sound, self.__ref_dBFS)
+            volume = self.__get_volume(pattern[1])
```

Comment:
set_volume and get_volume is a bit confusing after eachother. maybe we can all get_volume gain = get_gain() or get_volume_gain()